src/components/EntryList.vue

```diff
@@ -74,6 +74,10 @@
 						}
 						break
 					case 9:  // TAB
+						const modifier = !evt.shiftKey ? 1 : -1;
```

Comment:
This line would read better without the extra negation, IMO.  Don't need to change it, just my 2 cents.

Reply:
Yeah sorry about that, habits are leaking in. I just like to put something truthy in the truth part of ternary so that it look more logical as in `const variable = (someCond) ? true : false;`

Reply:
Also adding a bang in front of a property is the ultimate cast to boolean operator ;)